server/test/services/export/CsvExporterTest.java

```diff
@@ -124,10 +129,22 @@ public void programCsv_noRepeatedEntities() throws Exception {
         testQuestionBank.getSampleQuestionsForAllTypes().get(QuestionType.CHECKBOX);
     MultiSelectQuestion multiSelectApplicantQuestion =
         getApplicantQuestion(checkboxQuestion.getQuestionDefinition()).createMultiSelectQuestion();
-    String multiSelectHeader =
-        CsvExporterService.pathToHeader(multiSelectApplicantQuestion.getSelectionPath());
-    assertThat(records.get(1).get(multiSelectHeader)).isEqualTo("[toaster, pepper_grinder]");
-    // Check link for uploaded file
+
```

Comment:
**question**: What would the lift be to also test "not an option in not in program version"? I see it's tested in the browser tests so that could be a fast follow instead of part of this PR.

I've started some refactors of the test logic that could help here, for the JSON test:
https://github.com/civiform/civiform/blob/32bc2499819f2986eb15db46a31bfc7bc4a1cb18/server/test/services/export/JsonExporterTest.java#L268

Reply:
Yes may be a fast followup.

Reply:
Added a testcase for it :)